openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -58,15 +57,19 @@ <h2 class="step__title">
                     <ol class="list list--fields response__submission__content">
                         {% for part in saved_response.answer.parts %}
                             <li class="submission__answer__part">
+                                <h5 class="submission__answer__part__text__title">{% trans "The prompt for this section" %}</h5>
                                 <article class="submission__answer__part__prompt">
-                                    <h2 class="submission__answer__part__prompt__title">{% trans "The prompt for this section." %}</h2>
-
                                     <div class="submission__answer__part__prompt__copy">
                                         {{ part.prompt.description|linebreaks }}
                                     </div>
                                 </article>
+
+
+
                                 <div class="field field--textarea submission__answer__part__text">
-                                    <label class="sr" for="submission__answer__part__text__{{ forloop.counter }}__{{ xblock_id }}">{% trans "Your response to the question above." %}</label>
```

Comment:
This actually introduces an a11y failure (did you run the a11y bok choy tests recently? I hope there is a test that fails with this change!). There is no longer a label for the textarea.

![image](https://cloud.githubusercontent.com/assets/484484/20901997/273874da-bb03-11e6-82b6-68a87e41ddcc.png)